### DISCUSSION TITLE
RFC: What would you like to see for the next Release?

### DISCUSSION THREAD

**Discussion by PascaleBeier** [PascaleBeier/bootstrap-validate#65] [Announcements]
I am picking up on bootstrap-validate again to bring in support for Bootstrap 5. It will be a complete rewrite.

A few notes:

- The next Release will be v3, and only work with Bootstrap 5, still unsure about Bootstrap 4 
- Every piece of configuration like class names should be configurable at runtime
- It will emit real events to capture validation events (#60, #59)
- It will have a more robust approach for regex handling (#57)

Open Questions:

- How to handle the different Bootstrap versions?
[....]

Feel free to chime in with your ideas!

**Comment by ROHITSALUNKE1998** [comment]
@PascaleBeier @cmckenzie6 

This library will greatly benefit my projects once the following bugs are fixed:
1. Currently, when I attempt to add the same regex rule for two different input fields, the error message only appears beneath the first input field, not the second.
2. Additionally, the regex expression does not work with the OR (|) symbol.

https://user-images.githubusercontent.com/27411630/233772051-81c73186-62ca-498f-b15c-5795855bfd91.mp4

**Comment by shashankitsoft** [comment]
@PascaleBeier This really looks like a great plugin, with easy and compact to write syntaxes. Unfortunately, it seems it doesn't support Bootstrap 5 yet, which is now a standard and 5.3.2 is currently its version (although in plugin's about summary you have mentioned BS5 also). 
Can we expect a release of this library for BS5 soon? Or you abandoned this idea?

**Reply by PascaleBeier** [reply]
@shashankitsoft Sorry for the late response. I left the employer that funded this project 2 years ago, and there are currently no plans of upgrading it.
